### PR TITLE
Fix wrong error codes & names

### PR DESCRIPTION
**Pull request by yah01**
/kind improvement

### CODE REVIEW

**Comment by yah01** [comment]
rerun ut

File: pkg/mq/msgstream/mq_msgstream.go [hunk 1]
```diff
@@ -797,7 +797,7 @@ func (ms *MqTtMsgStream) Seek(msgPositions []*msgpb.MsgPosition) error {
 		if err != nil {
 			log.Warn("Failed to seek", zap.String("channel", mp.ChannelName), zap.Error(err))
 			// stop retry if consumer topic not exist
-			if errors.Is(err, mqwrapper.ErrTopicNotExist) {
+			if errors.Is(err, mqwrapper.ErrMqTopicNotExist) {
```
Comment (congqixia) [inline pkg/mq/msgstream/mq_msgstream.go:800]: `mqwrapper` or `merr`?

File: pkg/mq/msgstream/mqwrapper/kafka/kafka_consumer.go [hunk 2]
```diff
@@ -236,7 +236,7 @@ func (kc *Consumer) CheckTopicValid(topic string) error {
 		switch v := err.(type) {
 		case kafka.Error:
 			if v.Code() == kafka.ErrUnknownTopic || v.Code() == kafka.ErrUnknownPartition || v.Code() == kafka.ErrUnknownTopicOrPart {
-				return merr.WrapErrTopicNotFound(topic, "topic get latest msg ID failed, topic or partition does not exists")
+				return merr.WrapErrMqTopicNotFound(topic, "topic get latest msg ID failed, topic or partition does not exists")
```
Comment (congqixia) [inline pkg/mq/msgstream/mqwrapper/kafka/kafka_consumer.go:239]: Hide the detail error, `ErrUnkownPartition` info will be lost

**Pull request merged**